css/src/style.scss

```diff
@@ -1153,6 +1132,16 @@ ol[dnd-list] {
 	}
 }
 
+/**
+	* timepicker fixes
+	*/
+
+.ui-timepicker-table td a {
```

Comment:
It seems to me that only `display: block` is needed here.

In fact, we should consider moving it to the core css.

Reply:
Would be good to have it in core, but until this is in core, we should keep the fix here.

Reply:
We need
```
.ui-timepicker-table td a {
   display: block;
   width: 14px;
}
```
here. The width is needed as well. With
```
padding: 8px 7px;
line-height: 12px;
```
(see https://github.com/nextcloud/server/blob/7d9e47543482acd430a18b482cd61bdd5eecc3e9/core/css/styles.scss#L1118-L1120) the element would not be quadratic and the blue circle would be an ellipse.

Reply:
Same goes for datepicker.

Reply:
Hm, datepicker seems to be more tricky. Setting the width to 12 px leads to problems for numbers with two digits. --> Reverting.

Reply:
Ok, datepicker highlight is now also circular.